backend/helm_client/public.go

```diff
@@ -0,0 +1,96 @@
+package helm_client
+
+// helm list
+func List(o *ListReleaseOptions) (*ListReleaseRsp, error) {
+	c := &baseClient{}
+	c.InitClient(o.Namespace)
```

Comment:
```suggestion
	c := CreatClient(o.Namespace)
```
This makes it more clear and less redundant